src/motion/functionality/dom.tsx

```diff
@@ -58,6 +58,28 @@ function filterValidProps(props: MotionProps) {
     return domProps
 }
 
+const buildHTMLProps = (
+    values: MotionValuesMap,
+    style: CSSProperties,
+    isStatic?: boolean,
+    isDrag?: boolean
+) => {
+    // The `any` isn't ideal but it is the type of createElement props argument
+    const props: any = {
+        style: buildStyleAttr(values, style, isStatic),
+    }
+
+    if (isDrag) {
+        // Disable text selection
+        props.style.userSelect = "none"
+
+        // Disable the ghost element when a user drags
+        props.draggable = false
```

Comment:
Why do we override draggable?

Reply:
We probably should have been doing this anyway but if you don't do this you get a weird ghost element